### PR TITLE
Update pose design

### PR DESCRIPTION
**Pull request by jivey**
for: https://github.com/openstax/unified/issues/2305

- Remove the condensed styles, we prefer the image background when embedded
- Update the card design to allow for a wider display, and to use newer button and checkbox styles
- Update "I Agree" button text to "Accept and continue"

image

### CODE REVIEW

**Review by mwvolo — APPROVED** [review]

File: config/locales/pl.yml [hunk 1]
```diff
@@ -436,16 +436,15 @@ pl:
...
       contract_acceptance_required: >-
         Aby kontynuować, zapoznaj się z treścią poniższych regulaminów
         i zaakceptuj je klikając „Zgadzam się” na końcu tej strony.
       contracts_changed_notice: >-
         Zapisy regulaminu %{contract_title} zmieniły się. Prosimy zapoznać się
         z nową wersją, a następnie kliknąć „Zgadzam się” na końcu tej strony.
       have_read_terms_and_agree: >-
-        Zapoznałem/am się z przedstawionymi powyżej regulaminami i zgadzam
-        się na zawarte w nich warunki.
+        Przeczytałem i zgadzam się z tymi warunkami użytkowania.
```
Comment (jivey) [inline config/locales/pl.yml:447]: @ewaforman 👋 Would you mind looking at this and let me know if it sounds right? I'm hoping it's translating "I have read and agree to these terms of use" in a natural way. Thank you!

Comment (ewaforman) [reply config/locales/pl.yml:447]: @jivey 👋 “Zapoznałem/am się z przedstawionymi powyżej regulaminami i zgadzam
się na zawarte w nich warunki” sounds good.
“Przeczytałem i zgadzam się z tymi warunkami użytkowania” doesn’t sound good, it is too colloquially. And verb “read” is only masculine, not as in first sentence masculine and feminine.

Comment (jivey) [reply config/locales/pl.yml:447]: Thanks for the 👀! I'll revert it back to the original. And I forgot to ask, but what do you think of "Zaakceptuj i kontynuuj" for "Accept and continue"?

Comment (ewaforman) [reply config/locales/pl.yml]: @jivey "Zaakceptuj i kontynuuj" is okay.

**Pull request merged**